src/core.h

```diff
@@ -144,33 +144,19 @@ int conf_get_csv(const struct conf *conf, const char *name,
 int conf_get_float(const struct conf *conf, const char *name, double *val);
 
 
-/*
- * Metric
- */
-
-struct metric {
-	/* internal stuff: */
-	struct tmr tmr;
-	mtx_t lock;
-	uint64_t ts_start;
-	bool started;
-
-	/* counters: */
-	uint32_t n_packets;
-	uint32_t n_bytes;
-	uint32_t n_err;
-
-	/* bitrate calculation */
-	uint32_t cur_bitrate;
-	uint64_t ts_last;
-	uint32_t n_bytes_last;
-};
```

Comment:
can you please make this a separate patch, moving "struct metric" to metric.c ?

Reply:
It is a separate commit. You mean a separate PR? --> https://github.com/baresip/baresip/pull/1895